tests/test_io.py

```diff
@@ -0,0 +1,217 @@
+import configparser
+import logging
+import os
+import shutil
+
+import pytest
+
+import py
+from bidict import bidict
+from edges_io import io
+
+LOAD_ALIASES = bidict(
+    {
+        "ambient": "Ambient",
+        "hot_load": "HotLoad",
+        "open": "LongCableOpen",
+        "short": "LongCableShorted",
+    }
+)
+
+LOGGING = logging.getLogger("edges-io")
+
+
+@pytest.fixture("module")
+def create_testdir(tmp_path_factory):
+    testDir = create_test_env(tmp_path_factory)
+    return testDir
+
+
+@pytest.fixture("module")
+def create_test_env(tmp_path_factory):
+    # Create an ideal observation file using tmp_path_factory
+    pthList = ["Spectra", "Resistance", "S11"]
+    s11List = [
+        "Ambient",
+        "AntSim3",
+        "HotLoad",
+        "LongCableOpen",
+        "LongCableShorted",
+        "ReceiverReading01",
+        "ReceiverReading02",
+        "SwitchingState01",
+        "SwitchingState02",
+    ]
+    a = tmp_path_factory.mktemp("Test_Obs")
+    b = a / "Receiver01_2020_01_01_010_to_200MHz"
+    b.mkdir()
+    d = b / "25C"
+    d.mkdir()
```

Comment:
I have a feeling (not sure) that you could do `d = b.mkdir("25C")` all in one line